rocAL/rocAL/source/meta_data/meta_node_flip.cpp

```diff
@@ -27,7 +27,7 @@ void FlipMetaNode::initialize()
     _src_width_val.resize(_batch_size);
     _flip_axis_val.resize(_batch_size);
 }
-void FlipMetaNode::update_parameters(MetaDataBatch* input_meta_data)
+void FlipMetaNode::update_parameters(MetaDataBatch* input_meta_data, bool segmentation)
```

Comment:
If we don't modify this bool variable,  please use const to define

Reply:
Resolved in the latest commit